Core/src/main/java/com/plotsquared/core/plot/PlotInventory.java

```diff
@@ -103,9 +103,12 @@ public void close() {
         this.open = false;
     }
 
-    public void setItem(int index, PlotItemStack item) {
+    public boolean setItem(int index, PlotItemStack item) {
```

Comment:
Nevertheless we could put javadocs and since here I guess?

Reply:
@\inheritDoc should do the job i guess?

Reply:
From where do you want to inherit it from? This method is not overridden or delegated via super. 👀

Reply:
Oh wait, wrong file, my bad lmao